ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -717,6 +723,11 @@ chainSelectionForBlock cdb@CDB{..} blockCache hdr punish = electric $ do
                    -> Bool
     followsLoEFrag LoEDisabled _ = True
     followsLoEFrag (LoEEnabled loe) frag =
+      -- REVIEW: This 'frag' argument is the suffix of a diff (with rollback ==
+      -- 0 in 'addToCurrentChain' and /= 0 in 'switchToAFork'), and this suffix
+      -- might not intersect with the LoE fragment, because 'frag' is anchored
+      -- somewhere on the current chain and LoE frag's tip might be older than
+      -- that.
```

Comment:
The comments in this commit are introduced to be immediately removed in the next commit. It would be better to just drop this commit.

Reply:
Indeed.